persistence/src/main/java/edu/unc/lib/dl/persist/services/transfer/FSToFSTransferClient.java

```diff
@@ -49,8 +50,8 @@
 
     private static final CopyOption[] COPY_NO_OVERWRITE = { COPY_ATTRIBUTES };
     private static final CopyOption[] COPY_ALLOW_OVERWRITE = { COPY_ATTRIBUTES, REPLACE_EXISTING };
-    private static final CopyOption[] MOVE_NO_OVERWRITE = { };
-    private static final CopyOption[] MOVE_ALLOW_OVERWRITE = { REPLACE_EXISTING };
+
+    private static final Logger log = LoggerFactory.getLogger(FSToFSTransferClient.class);
 
     public FSToFSTransferClient(IngestSource source, StorageLocation destination) {
```

Comment:
You will also need to update the `StreamToFSTransferClient` with the roughly same safe write procedure.